setup.py

```diff
@@ -20,42 +20,6 @@ def finalize_options(self):
     def run(self):
         os.system('rm -vrf ./build ./dist ./*.pyc ./*.tgz ./*.egg-info')
 
-def get_dnsmasq():
-    """
-    Try to install dnsmasq on host machine if not present
-
-    :return: None
-    :rtype: None
-    """
-
-    if not os.path.isfile("/usr/sbin/dnsmasq"):
-        install = raw_input(("[" + constants.T + "*" + constants.W + "] dnsmasq not found " +
-                             "in /usr/sbin/dnsmasq, " + "install now? [y/n] "))
-
-        if install == "y":
-            if os.path.isfile("/usr/bin/pacman"):
-                os.system("pacman -S dnsmasq")
-            elif os.path.isfile("/usr/bin/yum"):
-                os.system("yum install dnsmasq")
-            else:
-                os.system("apt-get -y install dnsmasq")
-        else:
-            sys.exit(("[" + constants.R + "-" + constants.W + "] dnsmasq " +
-                      "not found in /usr/sbin/dnsmasq"))
-
-    if not os.path.isfile("/usr/sbin/dnsmasq"):
-        dnsmasq_message = ("\n[" + constants.R + "-" + constants.W +
-                           "] Unable to install the \'dnsmasq\' package!\n" + "[" + constants.T +
-                           "*" + constants.W + "] This process requires a persistent internet " +
-                           "connection!\nPlease follow the link below to configure your " +
-                           "sources.list\n" + constants.B + "http://docs.kali.org/general-use/" +
-                           "kali-linux-sources-list-repositories\n" + constants.W + "[" +
-                           constants.G + "+" + constants.W + "] Run apt-get update for changes " +
-                           "to take effect.\n" + "[" + constants.G + "+" + constants.W + "] " +
-                           "Rerun the script to install dnsmasq.\n[" + constants.R + "!" +
-                           constants.W + "] Closing")
-
-        sys.exit(dnsmasq_message)
```

Comment:
What I propose here if "dnsmasq" is missing, is that we print a message. Something like:
            `sys.exit(("[" + constants.R + "-" + constants.W + "] dnsmasq not found in /usr/sbin/dnsmasq"))` is sufficient.

Reply:
I disagree. I don't think we should do it in the `setup.py` file. The output will get lost with the installation and it is hard for people to see. I think we should only check it on run time.

Reply:
Let's print it at the end of the installation process then. Instead of printing the ASCII art which is an indication of a successful installation, we can print exactly what went wrong.